Apps/Src/ActivateArray.c

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "ActivateArray.h"
+
+void Task_ArrayConnection(void *p_arg) {
```

Comment:
What if the driver turns the key back (into the off position) after turning it on?